mlrun/utils/notifications/notification_pusher.py

```diff
@@ -440,6 +446,162 @@ def _update_notification_status(
         )
 
 
+class WorkflowNotificationPusher(NotificationPusher):
```

Comment:
Workflow is a shared name for remote and kfp, perhaps rename this class to 
```suggestion
class KFPNotificationPusher(NotificationPusher):
```
as its usage is by kfp only, right?

Reply:
Yes